app/serializers/plant_serializer.rb

```diff
@@ -0,0 +1,12 @@
+class PlantSerializer
```

Comment:
Lets include relationships here at the bottom.
`has_many :garden_plants
  has_many :gardens, through: :garden_plants`